pkg/api/types.go

```diff
@@ -2121,6 +2121,16 @@ type PodSpec struct {
 	// file if specified. This is only valid for non-hostNetwork pods.
 	// +optional
 	HostAliases []HostAlias
+	// If specified, indicates the pod's priority. "system" is a special keyword
+	// which indicates the highest priority. Any other name must be defined by
+	// creating a PriorityClass object with that name.
+	// If not specified, the pod priority will be zero.
+	// +optional
+	PriorityClassName string
+	// The priority value which is resolved by the Admission Controller from
```

Comment:
As someone who reads the API, I'm seeing that this is an optional field, but validation won't allow any value to be set.  How does the admission controller set the value?